bus-mapping/src/circuit_input_builder.rs

```diff
@@ -1175,7 +1228,7 @@ mod tracer_tests {
                 builder,
                 tx,
                 tx_ctx: TransactionContext::new(&block.eth_tx),
-                step: ExecStep::new(geth_step, 0, GlobalCounter(0), 0),
+                step: ExecStep::new(geth_step, 0, RWCounter(0), 0),
```

Comment:
This seems not updated, tho it doesn't affect the testing result.
```suggestion
                step: ExecStep::new(geth_step, 0, RWCounter::new(), 0),
```